### ISSUE TITLE
atom hangs when opening this ttcn3 file

### ISSUE THREAD

**Issue by pespin** [karolwiniarski/language-ttcn-3#3]
Using:
```
$ atom --version
Atom    : 1.42.0
Electron: 4.2.12
Chrome  : 69.0.3497.128
Node    : 10.11.0

language-ttcn3-3 : 0.1.3
```

I can open all other TTCN3 files from [1] correctly, yet opening this file [2] makes the atom process take full CPU and hang until it allows to be killed through GUI message a few seconds later.

I attach to this ticket the problematic file.
[PCU_Tests.ttcn.zip](https://github.com/karolwiniarski/language-ttcn-3/files/4010216/PCU_Tests.ttcn.zip)

I can open the file fine with other editors such as geany, less, vim, etc.
Some trial and error changing file content suggested the issue seems to be related to some bit/hex string in the file.

[1] https://git.osmocom.org/osmo-ttcn3-hacks/
[2] https://git.osmocom.org/osmo-ttcn3-hacks/tree/pcu/PCU_tests.ttcn

**Comment by pespin** [comment]
Still happening with:

```
Atom    : 1.45.0
Electron: 4.2.12
Chrome  : 69.0.3497.128
Node    : 10.11.0
```

**Comment by pespin** [comment]
I confirm that if I disable the "language-ttcn3" package then I can open the file with no issue, so the fault seems to be released to this module.

**Comment by pespin** [comment]
Smaller file to reproduce the issue. Use a file with this content:
```
module PCU_Tests {
        40240080400000000000000077628
}
```
Then add any letter "a" to "f" to the end of the character string, and Atom will hang. It hangs too if a cap letter is used ("A" to "F").

It doesn't hang if you add a character containing a number.

**Comment by karolwiniarski** [comment]
Please check a new version and let me know if problem is fixed.

**Comment by pespin** [comment]
Hi, seems fixed now with 0.3.0. Thanks!

**Comment by karolwiniarski** [comment]
Great!
